<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewQuestion</class>
 <widget class="QDialog" name="NewQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="answer1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer 4</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="question">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Correct:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="correctBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createQuestionButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
